### DISCUSSION TITLE
Install Current Ifcopenshell version

### DISCUSSION THREAD

**Discussion by Theinformatiker** [IfcOpenShell/IfcOpenShell#2795] [Q&A] [answered]
Hi all, I have a quick question. How is it possible to use the current version of ifcopenshell on VSCode and how can I upgrade it weekly. Normally I use PyCarm and with pip installed ifcopenshell but this is not the current vesion of ifcopenshell.
My idea was to download the main source folder from git and copy it to the : C:\Users\UserVk\AppData\Local\Programs\Python\Python310\Lib\site-packages
and then use it with import ifcopenshell.
But its not working :-(.
Anyone an idea of this problem

**Comment by Moult** [comment, marked as answer]
All the options you have available are here:

 - https://blenderbim.org/docs-python/ifcopenshell-python/installation.html
 - https://blenderbim.org/docs-python/ifcopenshell/installation.html (if you want to compile yourself)

You can use git for all the Python portion of ifcopenshell and simply copy over the binary blobs and generated ifcopenshell_wrapper* files from the latest build.

**Comment by Theinformatiker** [comment]
Thank you for your help
The easiest way to do this is with PyCharm and the command pip install ifcopenshell.
 However, this is not the latest version of the API, is it possible to update it?

**Reply by Moult** [reply]
You can use git to clone the latest IfcOpenShell from this repo, then copy over the `ifcopenshell_wrapper.py` and `_ifcopenshell_wrapper.so` (or whatever the equivalent is for your operating system) from the pip installation. That will keep you always at the latest version.
